### PR TITLE
gad-4: resolved issues in drafts: attachment was not visible, draft was not

### PR DESCRIPTION
**Pull request by prakhar-da-gama**
…saving without attachment

## Issue that this pull request solves

Closes: # (issue number)

## Types of changes

_Put an `x` in the boxes that apply_

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

-   [ ] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [ ] I have performed a self-review of my own code
-   [ ] I have created new branch for this pull request
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generate no new warnings
-   [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Review by MegaDestro — COMMENTED** [review]
Approved
Ping @akshatnema

File: FusionIIIT/applications/filetracking/views.py [hunk 1]
```diff
@@ -723,3 +668,12 @@ def forward_inward(request,id):
...
 
+
+
+
+# todo_items = []
+# from fastapi import FastAPI
+# app = FastAPI()
+# @app.get("/health")
+# def health():
+#     return {"status": "ok"}
```
Comment (akshatnema) [inline FusionIIIT/applications/filetracking/views.py]:
```suggestion
```

File: FusionIIIT/applications/filetracking/views.py [hunk 2]
```diff
@@ -109,16 +93,27 @@ def filetracking(request):
...
             if 'send' in request.POST:
                 uploader = request.user.extrainfo
+                print("***********************************************")
+                print(uploader)
                 subject = request.POST.get('title')
                 description = request.POST.get('desc')
                 design = request.POST.get('design')
+               # holdsdesignations = HoldsDesignation.get('user','working','designation').all()
+                print("***********************************************")
+                print(design)
```
Comment (akshatnema) [inline FusionIIIT/applications/filetracking/views.py]:
```suggestion
```

**Review by akshatnema — CHANGES_REQUESTED** [review]
@prakhar-da-gama please remove unnecessary comments from this PR.

File: FusionIIIT/applications/filetracking/views.py [hunk 3]
```diff
@@ -168,38 +167,37 @@ def filetracking(request):
...
     context = {
         'file': file,
         'extrainfo': extrainfo,
         'holdsdesignations': holdsdesignations,
         'designations': designations,
+        'all_recievers': all_recievers,
+        
+
+
+
```
Comment (akshatnema) [inline FusionIIIT/applications/filetracking/views.py]:
```suggestion
```

**Review by akshatnema — APPROVED** [review]

**Pull request merged**